### ISSUE TITLE
How to use ADTag with docker one-line runner code????????

### ISSUE THREAD

**Issue by TabPaD** [9seconds/mtg#36]
So as the title shows I'm using the docker image and the one-line runner code you've provided to start my proxy:
`$ docker run --name mtg --restart=unless-stopped -p 3128:3128 -p 3129:3129 -d nineseconds/mtg:stable my-defined-secret
[mtg-error-log.txt](https://github.com/9seconds/mtg/files/2381510/mtg-error-log.txt)
`
in this case the proxy starts with absolutely no problems.
The problem is that you haven't provided any information about how to implement adtag in this one-line runner code.
I tried to add my Adtag to the end of the code but then the the proxy server wouldn't start at all.
`$ docker run --name mtg --restart=unless-stopped -p 3128:3128 -p 3129:3129 -d nineseconds/mtg:stable my-defined-secret my-adtag`

**Comment by p-hash** [comment]
Can confirm: adtag doesnt work.   
`mtg` crashes with code 2 when you provide adtag argument

**Comment by 9seconds** [comment]
Weird, this is another problem, not related to adtag. I'll take a look, thanks!

**Comment by 9seconds** [comment]
Yep, seems that Telegram has changed an output of their API call and I have to rework this part of the code a little bit. Gonna do it tomorrow.

**Comment by TabPaD** [comment]
> 
> 
> Yep, seems that Telegram has changed an output of their API call and I have to rework this part of the code a little bit. Gonna do it tomorrow.

well I'm happy to hear that you're fixing the issue as I really like the smoothness of your bullshit-free mtproto proxy.

**Comment by 9seconds** [comment]
@TabPaD @p-hash please check. Fix has been released as a stable release v0.13, `stable` and `master` branches.

**Comment by p-hash** [comment]
@9seconds works fine, thanks a lot for bullshit free proxy

**Comment by TabPaD** [comment]
I confirm it's working fine now. Thanks..
